Patch to set various paths etc. for FHS, Debian policy etc. and to point
bug reporting URL to Debian.

diff --git a/doc/elinks.1.txt b/doc/elinks.1.txt
index 231b879..1dc82a6 100644
--- a/doc/elinks.1.txt
+++ b/doc/elinks.1.txt
@@ -141,9 +141,10 @@ is in state of beta testing.
 BUGS
 ----
 
-Please report any other bugs you find to the either the ELinks mailing list at
-mailto:elinks-users@linuxfromscratch.org[] or if you prefer enter them into
-http://bugzilla.elinks.cz/[the bug tracking system]. More information about
+Please report any bugs you find to the
+http://bugs.debian.org/src:elinks[Debian bug tracking] system using the
+reportbug program, or if you prefer enter them into the upstream elinks 
+http://bugzilla.elinks.or.cz/[bug tracking system]. More information about
 how to get in contact with developers and getting help can be found on
 http://elinks.cz/community.html[the community page].
 
diff --git a/doc/man/man5/elinks.conf.5 b/doc/man/man5/elinks.conf.5
index 62a8ea2..9a8c8b3 100644
--- a/doc/man/man5/elinks.conf.5
+++ b/doc/man/man5/elinks.conf.5
@@ -1128,7 +1128,7 @@ mime\.mailcap\.enable [0|1] (default: 1)
 Enable mailcap support\.
 .RE
 .PP
-mime\.mailcap\.path <str> (default: "~/\.mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap:/usr/share/mailcap:/usr/share/misc/mailcap")
+mime\&.mailcap\&.path <str> (default: "~/\&.mailcap:/etc/mailcap:/usr/local/etc/mailcap:/usr/share/mailcap:/usr/share/misc/mailcap")
 .RS 4
 Mailcap search path\. Colon\-separated list of files\. Leave as "" to use MAILCAP environment variable instead\.
 .RE
@@ -1170,7 +1170,7 @@ mime\.mimetypes\.enable [0|1] (default: 1)
 Enable mime\.types support\.
 .RE
 .PP
-mime\.mimetypes\.path <str> (default: "~/\.mime\.types:/etc/mime\.types:/usr/etc/mime\.types:/usr/local/etc/mime\.types:/usr/share/mime\.types:/usr/share/misc/mime\.types")
+mime\&.mimetypes\&.path <str> (default: "~/\&.mime\&.types:/etc/mime\&.types:/usr/local/etc/mime\&.types:/usr/share/mime\&.types:/usr/share/misc/mime\&.types")
 .RS 4
 The search path for mime\.types files\. Colon\-separated list of files\.
 .RE
diff --git a/src/setup.h b/src/setup.h
index 7ac026b..5f9e1c8 100644
--- a/src/setup.h
+++ b/src/setup.h
@@ -12,7 +12,7 @@
 
 #define ELINKS_HOMEPAGE			"http://elinks.cz/"
 #define ELINKS_DOC_URL			"http://elinks.cz/documentation/"
-#define ELINKS_BUGS_URL			"http://bugzilla.elinks.cz/"
+#define ELINKS_BUGS_URL			"http://bugs.debian.org/src:elinks"
 #define ELINKS_GITWEB_URL		"http://repo.or.cz/w/elinks.git"
 
 #define ELINKS_SOCK_NAME		"socket"
@@ -81,13 +81,14 @@
 #define DEFAULT_PAGER_PATH		"/usr/bin/pager"
 #define DEFAULT_LESS_PATH		"/usr/bin/less"
 #define DEFAULT_MORE_PATH		"/usr/bin/more"
-#define DEFAULT_MAILCAP_PATH		"~/.mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap:/usr/share/mailcap:/usr/share/misc/mailcap"
-#define DEFAULT_MIMETYPES_PATH		"~/.mime.types:/etc/mime.types:/usr/etc/mime.types:/usr/local/etc/mime.types:/usr/share/mime.types:/usr/share/misc/mime.types"
+#define DEFAULT_MAILCAP_PATH		"~/.mailcap:/etc/mailcap:/usr/local/etc/mailcap:/usr/share/mailcap:/usr/share/misc/mailcap"
+#define DEFAULT_MIMETYPES_PATH		"~/.mime.types:/etc/mime.types:/usr/local/etc/mime.types:/usr/share/mime.types:/usr/share/misc/mime.types"
+
 
 /* Default external commands (see osdep/newwin.c and/or system-specific osdep/
  * files) */
 #define DEFAULT_TWTERM_CMD		"twterm -e"
-#define DEFAULT_XTERM_CMD		"xterm -e"
+#define DEFAULT_XTERM_CMD		"x-terminal-emulator -e"
 #define DEFAULT_SCREEN_CMD		"screen"
 #define DEFAULT_OS2_WINDOW_CMD		"cmd /c start /c /f /win"
 #define DEFAULT_OS2_FULLSCREEN_CMD	"cmd /c start /c /f /fs"
